bash/config.sh

```diff
@@ -105,6 +101,22 @@ function import_public {
   psql $BUILD_ENGINE -v ON_ERROR_STOP=1 -q -f $target_dir/$name.sql
 }
 
+function import_qaqc_historic {
+  target_dir=$(pwd)/.library/qaqc/$VERSION
+  qaqc_do_path=spaces/edm-publishing/db-developments/main/latest/output/qaqc_historic.sql
```

Comment:
just noting this is broken because there's no historic file in main - works with your branch though

Reply:
It's in there, did a manual upload after the PR to generate those tables was approved